Files/ViewModels/ItemViewModel.cs

```diff
@@ -1117,6 +1117,16 @@ private async void RapidAddItemsToCollectionAsync(string path, string previousDi
                 IsLoadingItems = false;
 
                 AdaptiveLayoutHelpers.PredictLayoutMode(folderSettings, this);
+
+                // Find and select README file
+                foreach (var item in filesAndFolders)
+                {
+                    if (item.ItemName.Contains("readme", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Should this be limited to markdown and text files?

Reply:
Then we'd have to check for other formats as well e.g. .rtf
And what a file named "readme" would be used else for in the first place?

Reply:
I'm not really sure, I guess we can always revisit this.